packages/native-federation-tests/tsconfig.lib.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "declaration": true,
+    "skipLibCheck": true,
+    "types": []
+  },
+  "include": ["**/*.ts"],
+  "exclude": ["jest.config.ts", "**/*.spec.ts", "**/*.test.ts"]
```

Comment:
Should we exclude also `tsup.config.ts`?

Reply:
It doesn't change the documentation, I believe how it works is it uses the entry points then other objects with generated type declarations, so in this case even though it's included it isn't referenced by an entry point. But I excluded it anyhow